src/pages/Login.tsx

```diff
@@ -0,0 +1,108 @@
+import { Form, Field } from "react-final-form";
+
+import { ToastContainer, toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+
+const Login = () => {
+  const user: { email: string; password: string } = {
+    email: "pathomphob.s@20scoops.net",
+    password: "20scoop",
+  };
+
+  const onSubmit = (values: { email: string; password: string }) => {
```

Comment:
@bbun0639 could we group `{ email: string; password: string }` here and above to some type